docs/specs/om/open_metrics_spec_2_0.md

```diff
@@ -216,42 +226,103 @@ MetricFamilies of type Info MUST have an empty Unit string.
+
+Negative threshold classic buckets MAY be used.
+
+Classic bucket values MAY have exemplars. The value of the exemplar MUST be within the classic bucket. Exemplars SHOULD be put into the classic bucket with the highest threshold. A classic bucket MUST NOT have more than one exemplar.
+
+##### Exponential buckets
+
+Histogram MetricPoints with exponential buckets MUST have a Schema value. The Schema is an 8 bit signed integer between -4 and 127. Schema values between -4 and 127 are also called Standard Schemas.
+
+For any Standard Schema n, the Histogram MetricPoint MAY contain positive, negative exponential buckets and a single zero bucket. It is valid to have no exponentual buckets at all.
+
+The boundaries of a positive or negative exponential bucket with index i MUST be calculated as follows (using Python syntax):
+
+The upper inclusive limit of a positive exponential bucket: `(2**2**-n)**i`
+
+The lower exclusive limit of a positive exponential bucket: `(2**2**-n)**(i-1)`
+
+The lower inclusive limit of a negative exponential bucket: `-((2**2**-n)**i)`
+
+The upper exclusive limit of a negative exponential bucket: `-((2**2**-n)**(i-1))`
+
+i is an integer number that MAY be negative.
+
+There are exceptions to the rules above concerning the largest and smallest finite values representable as a float64 (called MaxFloat64 and MinFloat64 in the following) and the positive and negative infinity values (+Inf and -Inf):
+
+The positive exponential bucket that contains MaxFloat64 (according to the boundary formulas above) has an upper inclusive limit of MaxFloat64 (rather than the limit calculated by the formulas above, which would overflow float64).
+
+The next positive exponential bucket (index i+1 relative to the bucket from the previous item) has a lower exclusive limit of MaxFloat64 and an upper inclusive limit of +Inf. (It could be called a positive exponential overflow bucket.)
+
+The negative exponential bucket that contains MinFloat64 (according to the boundary formulas above) has a lower inclusive limit of MinFloat64 (rather than the limit calculated by the formulas above, which would underflow float64).
+
+The next negative exponential bucket (index i+1 relative to the bucket from the previous item) has an upper exclusive limit of MinFloat64 and an lower inclusive limit of -Inf. (It could be called a negative exponential overflow bucket.)
+
+Exponential buckets beyond the +Inf and -Inf buckets described above MUST NOT be used.
+
+If the zero bucket is present, the Historam MetricPoint MUST have a Zero threshold. The Zero threshold MUST be a non-negative float64 value (threshold >= 0.0). The boundaries of the Zero native bucket are `[-threshold, threshold]` inclusive.
```

Comment:
Maybe "inclusively" is better grammatically?

Reply:
done